newrelic/resource_newrelic_notifications_destination.go

```diff
@@ -69,11 +74,45 @@ func resourceNewRelicNotificationDestination() *schema.Resource {
 			"auth": {
 				Type:        schema.TypeMap,
 				Optional:    true,
-				ForceNew:    true,
+				ForceNew:    false,
 				Description: "A set of key-value pairs to represent a Notification destination auth.",
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				Sensitive:   true,
 			},
+			"active": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
```

Comment:
Same as channels. We should make this a normal value that a customer can change since they might want to disable it from time to time. 

Essentially anywhere we a mutation accepts an input field, the schema attribute will need to be a normal, non-computed value.